extensions/amp-sidebar/1.0/component.js

```diff
@@ -191,44 +191,49 @@ export function SidebarToolbar({
   ...rest
 }) {
   const ref = useRef();
-  const [matches, setMatches] = useState(false);
-  const [targetEl, setTargetEl] = useState(null);
+  const [target, setTarget] = useState();
 
   useEffect(() => {
-    const window = ref.current?.ownerDocument?.defaultView;
-    if (!window) {
-      return;
-    }
-
-    const mediaQueryList = window.matchMedia(mediaQueryProp);
-    const updateMatches = () => setMatches(mediaQueryList.matches);
-    mediaQueryList.addEventListener('change', updateMatches);
-    setMatches(mediaQueryList.matches);
-    return () => mediaQueryList.removeEventListener('change', updateMatches);
-  }, [mediaQueryProp]);
+    const selector = `#${CSS.escape(toolbarTarget)}`;
```

Comment:
Should this be `escapeCssSelectorIdent`?

Reply:
Good recommendation, updated in the latest revision! https://github.com/ampproject/amphtml/pull/34158/commits/d1c5a3eb1d7faf164c13ae9c7b13a0b2ac9dc0ed